src/main/java/sirius/biz/jdbc/storage/StorageController.java

```diff
@@ -167,14 +167,12 @@ private void applyQuery(String query, SmartQuery<VirtualObject> baseQuery, boole
         if (!(queryString.startsWith("*") || queryString.startsWith("/"))) {
             queryString = "/" + queryString;
         }
-
-        if (queryString.contains("*")) {
-            baseQuery.where(OMA.FILTERS.or(OMA.FILTERS.like(VirtualObject.PATH).matches(queryString).build(),
-                                           OMA.FILTERS.like(VirtualObject.OBJECT_KEY).matches(query).build()));
-        } else {
-            baseQuery.where(OMA.FILTERS.or(OMA.FILTERS.eq(VirtualObject.PATH, queryString),
-                                           OMA.FILTERS.eq(VirtualObject.OBJECT_KEY, query)));
+        if (!queryString.contains("*")) {
```

Comment:
this might also change the behaviour outside of the autocomplete ?

Reply:
hatten wir nicht prefix gesagt oder startsWith oder wie das heisst

Reply:
ja nur beim autocomplete und da startWith oder prefix oder wie das hiess